server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverPushNotificationActor.cs

```diff
@@ -105,7 +105,9 @@ private PushNotification CreatePushNotification(CalendarEventAssignedWithAdditio
             var content = new PushNotificationContent
             {
                 Title = this.pushNotificationConfig.Title,
-                Body = string.Format(this.pushNotificationConfig.Body, message.Event.Type, message.Owner.Name),
+                Body = this.pushNotificationConfig.Body
```

Comment:
@santee, do we also need dates in push notifications?

Reply:
nah